app/containers/ItunesContainer/index.js

```diff
@@ -35,6 +38,29 @@ const Container = styled.div`
     padding: ${(props) => props.padding}px;
   }
 `;
+const CustomisedT = styled(T)`
+  && {
+    color: ${colors.textColor};
+  }
+`;
+
+const BottomContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: ${(props) => props.maxwidth}px;
+  width: 90%;
+  margin: 10px auto;
+  padding: 5px 50px;
+  background-color: ${colors.mainColor};
+  border-radius: 10px;
+`;
+const MusicGrid = styled.div`
+  display: grid;
```

Comment:
Cool that you went with the grid layout. This will never be responsive though, You can get auto sized column with a few changes. https://css-tricks.com/auto-sizing-columns-css-grid-auto-fill-vs-auto-fit/
Do try this out!